### PR TITLE
Fixes to rpcserver error handling

### PR DESCRIPTION
**Pull request by bendikro**

### CODE REVIEW

File: deluge/core/rpcserver.py [hunk 1]
```diff
@@ -260,44 +270,54 @@ def send_error():
...
-                    def on_success(result):
+                return
+
+        log.debug("RPC dispatch %s", method)
+        try:
+            method_auth_requirement = self.factory.methods[method]._rpcserver_auth_level
+            auth_level = self.factory.authorized_sessions[self.transport.sessionno][0]
+            if auth_level < method_auth_requirement:
+                # This session is not allowed to call this method
+                log.debug("Session %s is trying to call a method it is not "
```
Comment (cas--) [inline deluge/core/rpcserver.py]: I would reword this to "Session %s is attempting an unauthorized method call!"

Comment (bendikro) [inline deluge/core/rpcserver.py]: Fixed

**Comment by cas--** [comment]
Merged, thanks :)
